docs/rules/experimental-require-slot-types.md

```diff
@@ -0,0 +1,113 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "svelte/experimental-require-slot-types"
+description: "require slot type declaration using the `$$Slots` interface"
+---
+
+# svelte/experimental-require-slot-types
+
+> require slot type declaration using the `$$Slots` interface
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> **_This rule has not been released yet._** </badge>
+
+## :book: Rule Details
+
+This rule enforces the presence of the `$$Slots` interface if any slots are present in the component. This interface declares all of the used slots and their props and enables typechecking both in the component itself as well as all components that include it.
+The  `$$Slots` interface is experimental and is documented in [svelte RFC #38](https://github.com/dummdidumm/rfcs/blob/ts-typedefs-within-svelte-components/text/ts-typing-props-slots-events.md#typing-slots).
+
+<ESLintCodeBlock>
+
+<!--eslint-skip-->
+
+```svelte
+<!-- ✓ GOOD -->
+<script>
+  // eslint svelte/experimental-require-slot-types: "error"
```

Comment:
Could you change it to use block comments?

```suggestion
  /* eslint svelte/experimental-require-slot-types: "error" */
```

https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments

Reply:
Of course :) But I think it should work either way ;)

Reply:
I think it only works with block comments.  Thanks for the change anyway.